modular_skyrat/modules/modular_vending/code/wardrobes.dm

```diff
@@ -135,3 +135,9 @@
 		/obj/item/clothing/suit/det_suit/cowboyvest = 2,
 		/obj/item/taperoll/police = 1,
 	)
+
+/obj/machinery/vending/wardrobe/law_wardrobe
+	skyrat_products = list(
+		/obj/item/clothing/under/rank/civilian/lawyer/beige/skirt/demon = 3,
+		/obj/item/clothing/under/rank/civilian/lawyer/beige/skirt/demon/beeze = 2,
```

Comment:
```suggestion
		/obj/item/clothing/under/rank/civilian/lawyer/beige/skirt/demon/beeze = 2,
		/obj/item/clothing/suit/toggle/lawyer/black/better = 2,
```

I'm lazy and don't want to wait for this pr to be published so I can do my "add shit to vendors you fuckers" pr

Reply:
hate u